structuredefinitions/UKCore-Observation-VitalSigns-BMI.xml

```diff
@@ -53,6 +40,10 @@
       <sliceName value="snomedCT" />
       <min value="1" />
       <max value="1" />
+      <binding>
+        <strength value="required" />
```

Comment:
As discussed earlier, swap these back to preferred, and raise as a comment during the calls?

Reply:
done